test/actor.test.ts

```diff
@@ -408,7 +408,7 @@ describe('actors', () => {
         initial: 'initial',
         states: {
           initial: {
-            entry: assign(() => ({
+            entry: assign((): { serverRef: Actor } => ({
```

Comment:
quite frankly I believe this shouldnt be needed but TS failed to "lose" information about the inferred type (Interpreter) when matching against expected one (Actor), even if Interpreter extends Actor